wrangler.toml

```diff
@@ -1,11 +1,16 @@
 # Top-level configuration
 name = "retromatch-game-info"
 main = "workers/scrape.ts"
-compatibility_date = "2023-12-20"
+compatibility_date = "2023-10-30"
 
+node_compat = true
 workers_dev = false
-route = { pattern = "retromatch/*", zone_name = "retromatch" }
+# route = { pattern = "retromatch/*", zone_name = "retromatch" }
```

Comment:
oh yeah I got an error when this wasn't commented

Reply:
Ack! Thanks for catching this. This was holding me from successfully deploying the worker